### DISCUSSION TITLE
Where can I find the source code for SeamlessWM?

### DISCUSSION THREAD

**Discussion by literate-goggles** [facebookresearch/seamless_communication#308] [General]
Hi, the original paper mentions SeamlessWM, but I couldn't find it in the source code and can't get if it is now applied after the main Unity2 generation pipeline or not, and how can I use the actual classifier?

**Comment by avidale** [comment]
The codebase for Seamless audio watermarking  was released as https://github.com/facebookresearch/audioseal. 
The actual classifier was not released, and most likely it will never go public.
